include/deal.II/lac/sparse_vanka.templates.h

```diff
@@ -113,13 +111,15 @@ SparseVanka<number>::compute_inverses()
 #else
   const size_type n_inverses =
     std::count(selected->begin(), selected->end(), true);
-
-  const size_type n_inverses_per_thread =
-    std::max(n_inverses / n_threads, static_cast<size_type>(1U));
+  // somewhat arbitrarily set up an equal number of tasks as we have threads
+  const std::size_t n_tasks = MultithreadInfo::n_threads();
```

Comment:
This would work substantially better if you had something like 4x or 8x the number of tasks.

Reply:
I agree completely but I would like to leave that for a followup PR. Is that okay?

Reply:
Yes.